fortanix-vme/fortanix-vme-abi/src/lib.rs

```diff
@@ -158,4 +3397,305 @@ mod test {
             panic!("Not IPv4")
         }
     }
+
+    #[test]
+    fn test_error() {
+        let data: Vec<(Error, Vec<u8>)> = Vec::from([
+            (Error::ConnectionNotFound, Vec::from([0x72, 0x43, 0x6f, 0x6e, 0x6e, 0x65, 0x63, 0x74, 0x69, 0x6f,
```

Comment:
An alternative approach to testing this, would be to duplicate the structs and enums defined above here in this `test` module and add a `derive(Serialize, Deserialize)` attribute to them and then test that the result of the "manually" implemented `Serialize`/`Deserialize` traits matches that of the derived ones.